### PR TITLE
PF-2388: expose getRegionInfo

### PR DESCRIPTION
**Pull request by yuhuyoyo**

### CODE REVIEW

**Comment by yuhuyoyo** [comment]
@mikenev

File: service/src/main/java/bio/terra/workspace/service/policy/TpsApiDispatch.java
Comment (mikenev) [inline service/src/main/java/bio/terra/workspace/service/policy/TpsApiDispatch.java:205]: Curious if the tests would also get cached data. Seems like they would. It shouldn't really matter since the data doesn't change that much.

**Review by mikenev — APPROVED** [review]

File: service/src/main/java/bio/terra/workspace/app/configuration/external/PolicyServiceConfiguration.java
Comment (ddietterich) [inline service/src/main/java/bio/terra/workspace/app/configuration/external/PolicyServiceConfiguration.java:17]: I am mistrustful of Spring caching. We tried to use it in TDR and it did not work well. We hand-coded instead. It looks like you have not configured a cache provider.

See [here](https://docs.spring.io/spring-boot/docs/2.1.6.RELEASE/reference/html/boot-features-caching.html#boot-features-caching-provider) for choices and how to configure. Also [here](https://docs.spring.io/spring-framework/docs/5.1.8.RELEASE/spring-framework-reference/integration.html#cache) for more info.

File: service/src/main/java/bio/terra/workspace/service/policy/TpsApiDispatch.java
Comment (ddietterich) [reply service/src/main/java/bio/terra/workspace/service/policy/TpsApiDispatch.java:205]: I wonder if this is a premature optimization. Do we know that we have a performance or load issue? Is WSM the right place to put caching or should it be in TPS? A TPS cache would have the advantage of knowing when to invalidate.

**Review by ddietterich — COMMENTED** [review]
The code is fine. Two things:
1. I'm not so sure we want caching as implemented and in WSM
2. I wonder if we should get the naming resolved before merging in this new API

File: service/src/main/java/bio/terra/workspace/app/configuration/external/PolicyServiceConfiguration.java
Comment (yuhuyoyo) [reply service/src/main/java/bio/terra/workspace/app/configuration/external/PolicyServiceConfiguration.java:17]: I'm using spring-boot-starter-cache so I don't need to register a cache provider.

"When using Spring Boot, the mere presence of the starter package on the classpath alongside the EnableCaching annotation would register the same ConcurrentMapCacheManager. So there is no need for a separate bean declaration."

[spring-boot-starter-cache](https://search.maven.org/search?q=g:org.springframework.boot%20a:spring-boot-starter-cache)

You suggested caching in another ticket hence I implemented it. Fine to not cache :)

File: openapi/src/common/schemas.yaml
Comment (yuhuyoyo) [reply openapi/src/common/schemas.yaml:317]: sure!

can't merge it anyway. Would you like me to change it?

**Review by ddietterich — APPROVED** [review]

**Pull request merged**
